src/cascadia/TerminalSettingsModel/GlobalAppSettings.cpp

```diff
@@ -342,3 +342,49 @@ winrt::Windows::Foundation::Collections::IMapView<winrt::hstring, winrt::Microso
 {
     return _commands.GetView();
 }
+
+// Method Description:
+// - Create a new serialized JsonObject from an instance of this class
+// Arguments:
+// - <none>
+// Return Value:
+// - the JsonObject representing this instance
+Json::Value GlobalAppSettings::ToJson() const
+{
+    Json::Value json{ Json::ValueType::objectValue };
+
+    // Exclude DebugFeatures from the list below

```

Comment:
I think the correct solution here is to add `debugFeaturesDefault` as the final fallback value like this:
```c++
// GlobalAppSettings.h

#ifdef _DEBUG
static constexpr bool debugFeaturesDefault{ true };
#else
static constexpr bool debugFeaturesDefault{ false };
#endif

GETSET_SETTING(bool, DebugFeaturesEnabled, debugFeaturesDefault);
```

But I think I recall @DHowett not wanting that `static constexpr` in the header file? (pinging him because I don't remember, exactly)

Reply:
There's something weird about constexprs in the header that I cannot totally recall at this moment. I'm having a vague memory of when I was trying to fix the constexpr static analysis warnings...

Reply:
Moved it back to the cpp and added a helper function. Sounds like we _know_ it's bad, but can't remember why hahaha.

Reply:
I'm unresolving this conversation because it never explained why we were skipping serializing it and what the user experience is for when they try to round-trip it.

If we are not serializing a setting, we had better put a damn good comment in the code explaining not just _that_ we're skipping it but _WHY_ we're skipping it.

Reply:
It's been a while since I looked at this. I'm adding it back in and removing the comment. Since we're having `debugFeaturesDefault` be the system set fallback value, no reason why we should be skipping it.